studies/forms.py

```diff
@@ -62,53 +79,24 @@ class Meta:
+            <p>For more information on study types and their metadata, please
+            <a href="https://lookit.readthedocs.io/en/develop/experimenter.html#editing-study-type">see the documentation.</a></p>'''
         }
-
-
-# Form for creating a new study. Set study_type and structure in same form
-# in this case, rather than separately in the build form.
-class StudyForm(StudyEditForm):
-    structure = forms.CharField(label='Build Study - Add JSON', widget=AceOverlayWidget(mode='json', wordwrap=True, theme='textmate', width='100%', height='100%', showprintmargin=False), required=False, help_text='Add the frames of your study as well as the sequence of those frames.  This can be added later.')
-
-    def clean_structure(self):
-        structure = self.cleaned_data['structure']
-        try:
-            json_data = json.loads(structure)  # loads string as json
-        except:
-            raise forms.ValidationError("Save failed due to invalid JSON! Please use valid JSON and save again. If you reload this page, all changes will be lost.")
-        return json_data
-
-    class Meta(StudyEditForm.Meta):
-        fields = StudyEditForm.Meta.fields + ['structure', 'study_type']
         
-        labels = StudyEditForm.Meta.labels.copy()
-        labels['study_type'] = 'Study Type'
-        
-        help_texts = StudyEditForm.Meta.help_texts.copy()
-        help_texts['study_type'] = "Specify the build process as well as the parameters needed by the experiment builder. If you don't know what this is, just select the default."
-
-
-class StudyUpdateForm(StudyEditForm):
-    structure = forms.CharField(label='Build Study - Add JSON',
-                                widget=AceOverlayWidget(mode='json', wordwrap=True, theme='textmate', width='100%',
-                                                        height='100%', showprintmargin=False), required=False,
-                                help_text='Add the frames of your study as well as the sequence of those frames.  This can be added later.')
-
-    def clean_structure(self):
-        structure = self.cleaned_data['structure']
-        try:
-            json_data = json.loads(structure)  # loads string as json
-        except:
-            raise forms.ValidationError("Save failed due to invalid JSON! Please use valid JSON and save again. If you reload this page, all changes will be lost.")
-        return json_data
-
-    class Meta(StudyEditForm.Meta):
-        fields = StudyEditForm.Meta.fields + ['structure']
-
-        labels = StudyEditForm.Meta.labels.copy()
-
-        help_texts = StudyEditForm.Meta.help_texts.copy()
+class StudyEditForm(StudyForm):
+    
+    class Meta(StudyForm.Meta):
+        help_texts = StudyForm.Meta.help_texts.copy()
+        help_texts['study_type'] += '''<p class="help-block"> Once you've filled in the required metadata, you'll have to build the dependencies for your 
```

Comment:
Do we need to have HTML in here? Let's keep HTML out of class members

Reply:
See below and https://docs.djangoproject.com/en/2.1/ref/models/fields/#help-text